gossip/src/cluster_info_metrics.rs

```diff
@@ -536,13 +536,11 @@ pub(crate) fn submit_gossip_stats(
         .pull
         .votes
         .into_iter()
-        .map(|(slot, num_votes)| (*slot, *num_votes))
```

Comment:
Is this change related to lru? looks a bit confusing

Reply:
Yes, the type of `CrdsDataStats.votes` is `LruCache<Slot, usize>`